src/service/ngJwtAuthService.ts

```diff
@@ -721,4 +721,20 @@ export class NgJwtAuthService {
         return this.retrieveAndProcessToken(endpoint, authHeader);
     }
 
+    /**
+     * @returns boolean
+     */
+    public tokenHasExpired():boolean {
+
+        this.tokenData = this.readToken(this.rawToken);
+
+        if (!this.rawToken) {
+            return false;
+        }        
+        
+        let expiryDate = moment(this.tokenData.data.exp * 1000);
+
+        return expiryDate < moment();
+    }
```

Comment:
Haha hey @zakhenry ! It is the first situation. Just a unique situation where I would like to know if the token has expired. Will refactor. Also I noticed Travis was playing up I think it was this 'The command "pr-bumper check" failed and exited with 1 during .', is there a special way to bump versions here?